views/reports/monthly_report.php

```diff
@@ -1,1248 +1,1985 @@
+				</td>
+			</tr>
+			<tr>
+				<td>
+					<b>
+               Native American
+               </b>
+				</td>
+				<td>
+					<?php if (isset($natAmMonthRes)) echo $natAmMonthRes; else echo "";?>
+				</td>
+				<td>
+					<?php if (isset($natAmNewRes)) echo $natAmNewRes; else echo "";?>
+				</td>
+				<td>
+					<?php if (isset($natAmDupRes)) echo $natAmDupRes; else echo "";?>
+				</td>
+				<td>
+					<?php if (isset($natAmYearRes)) echo $natAmYearRes; else echo "";?>
+				</td>
+			</tr>
+			<tr>
+				<td>
+					<b>
+               Other
+               </b>
+				</td>
+				<td>
+					<?php if (isset($otherRacMonthRes)) echo $otherRacMonthRes; else echo "";?>
+				</td>
+				<td>
+					<?php if (isset($otherRacNewRes)) echo $otherRacNewRes; else echo "";?>
+				</td>
+				<td>
+					<?php if (isset($otherRacDupRes)) echo $otherRacDupRes; else echo "";?>
+				</td>
+				<td>
+					<?php if (isset($otherRacYearRes)) echo $otherRacYearRes; else echo "";?>
+				</td>
+			</tr>
+		</tbody>
+	</table>
+	<br />
+	<h5 class="text-center">
+		Zip Code Data
+	</h5>
+	<h6 class="text-center">
+		For use by PSP, ISP, TLC, & CAC only
+	</h6>
+	<table class="table table-hover">
+		<thead>
+			<tr>
+				<th>
+					Zip
+				</th>
+				<th>
+					Town
+				</th>
+				<th>
+					Current Month
+				</th>
+				<th>
+					Newly Served
+				</th>
+				<th>
+					Duplicate Served
+				</th>
+				<th>
+					YTD
+				</th>
+			</tr>
+		</thead>
+		<tbody>
+			<tr>
+				<td>
+					<b>
+               12501 
+               </b>
+				</td>
+				<td>
+					<b>
+               Amenia
+               </b>
+				</td>
+				<td>
+					<?php if (isset($_12501MonthRes)) echo $_12501MonthRes; else echo "";?>
+				</td>
+				<td>
+					<?php if (isset($_12501NewRes)) echo $_12501NewRes; else echo "";?>
+				</td>
+				<td>
+					<?php if (isset($_12501DupRes)) echo $_12501DupRes; else echo "";?>
+				</td>
+				<td>
+					<?php if (isset($_12501YearRes)) echo $_12501YearRes; else echo "";?>
+				</td>
+			</tr>
+			<tr>
+				<td>
+					<b>
+               12504
+               </b>
+				</td>
+				<td>
+					<b>
+               Annandale
+               </b>
+				</td>
+				<td>
+					<?php if (isset($_12504MonthRes)) echo $_12504MonthRes; else echo "";?>
+				</td>
+				<td>
+					<?php if (isset($_12504NewRes)) echo $_12504NewRes; else echo "";?>
+				</td>
+				<td>
+					<?php if (isset($_12504DupRes)) echo $_12504DupRes; else echo "";?>
+				</td>
+				<td>
+					<?php if (isset($_12504YearRes)) echo $_12504YearRes; else echo "";?>
+				</td>
+			</tr>
+			<tr>
+				<td>
+					<b>
+               12506
+               </b>
+				</td>
+				<td>
+					<b>
+               Bangall
+               </b>
+				</td>
+				<td>
+					<?php if (isset($_12506MonthRes)) echo $_12506MonthRes; else echo "";?>
+				</td>
+				<td>
+					<?php if (isset($_12506NewRes)) echo $_12506NewRes; else echo "";?>
+				</td>
+				<td>
+					<?php if (isset($_12506DupRes)) echo $_12506DupRes; els
```

Comment:
You can put this in the `custom.css` file instead of amending it to each file.

Reply:
I'm assuming you're talking about the @media print css. I'll put the overflow and navbar hiding, but do we want to hide form-groups and buttons by default as well? If not, I can leave that part here in this specific files.